<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LevantWidget</class>
 <widget class="QWidget" name="LevantWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LevantWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="baseStatsBox">
     <property name="font">
      <font>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>Base Stats</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QCheckBox" name="baseStatsRandom">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Random</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="animationBox">
     <property name="title">
      <string>Animation</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QCheckBox" name="animationBetaUsingItem">
        <property name="text">
         <string>Beta Using Item</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="animationRandomWeaponsPosture">
        <property name="text">
         <string>Random Weapons Posture</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
